Dockerfile.build

```diff
@@ -0,0 +1,30 @@
+FROM node:11.15.0 AS run
```

Comment:
FAR: Maybe rename Dockerfile.build with Dockerfile as suggested by @St0rmingBr4in and keep the different steps. Advantage: Dockerfile is default name.